### PR TITLE
Fix: prevent storybook form crashing when all values are removed in multiselect.

### PR DESCRIPTION
**Pull request by mckn**
**Special notes for your reviewer**:
Just added a small check to prevent the SB from crashing when all values are removed from multiselect.

### CODE REVIEW

**Comment by tskarhed** [comment]
This might be something we want to handle in SelectBase actually, since it is supposed to "work" that way.
https://github.com/JedWatson/react-select/issues/3632

**Review by dprokop — APPROVED** [review]
🥇

**Comment by mckn** [comment]
> This might be something we want to handle in SelectBase actually, since it is supposed to "work" that way. Otherwise I'm certain we're going to have more issues where it crashes.
> [JedWatson/react-select#3632](https://github.com/JedWatson/react-select/issues/3632)

Awesome, feedback! I will fix.

File: packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx [hunk 1]
```diff
@@ -178,6 +178,15 @@ export function SelectBase<T>({
 }: SelectBaseProps<T>) {
   const theme = useTheme();
   const styles = getSelectStyles(theme);
+  const onChangeWithEmpty = useCallback(
+    (value: SelectValue<T>) => {
+      if (isMulti && !Array.isArray(value)) {
+        return onChange([]);
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx]: Shouldn't the value be in the call?
```suggestion
        return onChange([value]);
```

Comment (mckn) [reply packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx]: ah, nope - because it is undefined or null in that scenario.

Comment (mckn) [reply packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx]: bit I guess I need to make that a bit clearer in the code.

Comment (hugohaggmark) [reply packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx]: maybe check for undefined instead?

Comment (mckn) [reply packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx]: yeah, I will update this.

Comment (mckn) [reply packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx]: good feedback :)

**Review by hugohaggmark — APPROVED** [review]
LGTM!

**Pull request merged**